src/languageService/kustoLanguageService.ts

```diff
@@ -1437,12 +1442,14 @@ class KustoLanguageService implements LanguageService {
             );
 
             // TODO: handle outputColumns (right now it doesn't seem to be implemented for any function).
-            return new sym.FunctionSymbol.$ctor16(fn.name, fn.body, parameters, null);
+            return new sym.FunctionSymbol.$ctor16(fn.name, fn.body, parameters, fn.docstring);
         };
 
         const createTableSymbol: (tbl: s.Table) => sym.TableSymbol = (tbl) => {
             const columnSymbols = tbl.columns.map((col) => KustoLanguageService.createColumnSymbol(col));
-            return new sym.TableSymbol.$ctor3(tbl.name, columnSymbols);
+            const symbol = new sym.TableSymbol.$ctor3(tbl.name, columnSymbols);
```

Comment:
can clusters / databases / columns have a docstring as well? we should support anyone that does.

Reply:
Added column support but the c# code doesn't support clusters and databases.